scripts/deployOPEPE.ts

```diff
@@ -3,16 +3,16 @@ import { UniV3TwapOracle, UniV3TwapOracle__factory } from "../typechain-types";
 
 async function main() {
   const oPepe = await ethers.deployContract("OErc20Delegator", [
-    "0x6982508145454Ce325dDbE47a25d4ec3d2311933",
-    "0x7D61ed92a6778f5ABf5c94085739f1EDAbec2800",
-    "0x4021047a36AC60b40316F630307cD4791cdAeD52",
-    0.2e9,
+    "0x6982508145454Ce325dDbE47a25d4ec3d2311933", // underlying token address (PEPE)
+    "0x7D61ed92a6778f5ABf5c94085739f1EDAbec2800", // Comptroller address
+    "0x4021047a36AC60b40316F630307cD4791cdAeD52", // Interest rate model address
+    0.2e9, // initial exchange rate mantissa
     "Onyx PEPE",
     "oPEPE",
-    8,
-    "0x08eDF0F2AF8672029eb445742B3b4072c6158DF3",
-    "0x9dcb6bc351ab416f35aeab1351776e2ad295abc4",
-    "0x"
+    8, // oToken decimals
+    "0x08eDF0F2AF8672029eb445742B3b4072c6158DF3", // admin address (Timelock)
```

Comment:
Set oPepeAdmin to `0x08eDF0F2AF8672029eb445742B3b4072c6158DF3` as const, and use in `await oPepeOracle.setAdmin()`

Reply:
Move all the constants to a separate file.

<img width="487" alt="image" src="https://github.com/Onyx-Protocol/onyx-protocol/assets/130896629/4fc0e031-c8c1-4a88-b62b-cb84287683e5">